.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--
+See http://loopback.io/doc/en/contrib/triaging-pull-requests.html to understand
+the MINIMUM requirements for submitting pull requests.
+-->
+
+### Type of change
+
+<!--
+See http://semver.org/ for more info.
+-->
+
+- [ ] MAJOR -- Incompatible API changes (ie. breaking change(s))
```

Comment:
`Backwards-incompatible changes`

1) Just "incompatible" does not say what we are not compatible with
2) It's not only about API, a change in behaviour requires SEMVER-MAJOR too.

I am also proposing to prefix all three items with `SEMVER-`, e.g. `SEMVER-MAJOR`.